addons/website_payment/static/src/interactions/website_payment_donation.js

```diff
@@ -0,0 +1,33 @@
+import { Interaction } from "@web/public/interaction";
+import { registry } from "@web/core/registry";
+
+export class WebsitePaymentDonation extends Interaction {
+    static selector = ".o_donation_payment_form";
+    dynamicContent = {
+        ".o_amount_input": { "t-on-focus": this.onFocusAmountInput },
+        "#donation_comment_checkbox": { "t-on-change": this.onChangeDonationComment },
+    }
+
+    /**
+     * @param {Event} ev
+     */
+    onChange(ev) {
+        const checked = ev.currentTarget.checked;
+        const donationCommentEl = this.el.querySelector('#donation_comment');
+        donationCommentEl.classList.toggle('d-none', !checked);
+        if (!checked) {
+            donationCommentEl.value = "";
+        }
+    }
+
+    onFocusAmountInput() {
+        const otherAmount = this.el.querySelector("#other_amount")
+        if (otherAmount) {
+            otherAmount.setAttribute("checked", true)
```

Comment:
`;`

Reply:
twice

Reply:
or just a one-liner `this.el.querySelector("#other_amount")?.setAttribute("checked", true);`